data/test/saves/backstory-0.6.0.0.yaml

```diff
@@ -0,0 +1,58 @@
+# obtained by:
+# git checkout 0.6.0.0
+# cabal run swarm -- --scenario Tutorials/Backstory --run <(echo 'say "Ready!"')
+path: Tutorials/backstory.yaml
+status:
+  contents:
+    - initialCode: /dev/fd/15
```

Comment:
lol, I am not sure what the right thing to do is here, but surely it is not useful to store `initialCode: /dev/fd/15` :laughing: 

(This is not a comment about this PR, but about what we store in `initialCode` in general)